src/main/java/lotto/domain/LastWeekWinnerNumber.java

```diff
@@ -2,12 +2,29 @@
 
 public class LastWeekWinnerNumber {
 	private final Lotto winnerNumber;
+	private final LottoNumber bonusNumber;
 
-	public LastWeekWinnerNumber(String inputNumber) {
-		winnerNumber = new Lotto(inputNumber);
+	public LastWeekWinnerNumber(String inputNumber, LottoNumber bonusNumber) {
+		this.winnerNumber = new Lotto(inputNumber);
+		if (this.winnerNumber.getGameNumberList().contains(bonusNumber)) {
+			throw new IllegalArgumentException("보너스 번호는 당첨번호와 다른 번호여야 합니다.");
+		}
```

Comment:
validation을 수행한 후 값을 할당하는게 적절한 것 같아요 :)
```suggestion
		if (this.winnerNumber.getGameNumberList().contains(bonusNumber)) {
			throw new IllegalArgumentException("보너스 번호는 당첨번호와 다른 번호여야 합니다.");
		}
		this.winnerNumber = new Lotto(inputNumber);
```